deploy/charts/mixer/templates/_helpers.tpl

```diff
@@ -0,0 +1,51 @@
+{{/*
+Expand the name of the chart.
```

Comment:
nit: would be great if the files came with a descriptive header comment.

Reply:
Done.